### PR TITLE
(fix) improve storage.get docs

### PR DESCRIPTION
**Pull request by sw-yx**
(fix) improve storage.get docs

*Issue #, if available:* closes https://github.com/aws-amplify/amplify-js/issues/847

*Description of changes:*

helps address valid concerns raised by @donnrri

### CODE REVIEW

**Comment by sw-yx** [comment]
can i get a quick 👀  @mauerbac or @dabit3

File: docs/lib/storage/fragments/js/download.md [hunk 1]
```diff
@@ -1,44 +1,86 @@
...
-Storage.get('test.txt', { expires: 60 })
-    .then(result => console.log(result))
-    .catch(err => console.log(err));
-```
\ No newline at end of file
+await Storage.get('filename.txt', { expires: 60 })
+```
+
+
+### Frequent User Questions
```
Comment (mauerbac) [inline docs/lib/storage/fragments/js/download.md:75]: User Issues?

File: docs/lib/storage/fragments/js/download.md [hunk 2]
```diff
@@ -1,44 +1,86 @@
...
+
+Users often run into unexpected issues, so we are giving you advance notice in documentation with links to open issues - please upvote what you need, to help the team prioritize.
```
Comment (mauerbac) [inline docs/lib/storage/fragments/js/download.md:77]: Users can run into

**Review by mauerbac — APPROVED** [review]
LGTM

File: docs/lib/storage/fragments/js/download.md (hunk 2 above)
Comment (sw-yx) [inline docs/lib/storage/fragments/js/download.md:77]:
```suggestion
Users can run into unexpected issues, so we are giving you advance notice in documentation with links to open issues - please upvote what you need, to help the team prioritize.
```

File: docs/lib/storage/fragments/js/download.md (hunk 1 above)
Comment (sw-yx) [inline docs/lib/storage/fragments/js/download.md:75]:
```suggestion
### Frequently Asked Questions
```

File: docs/lib/storage/fragments/js/download.md [hunk 3]
```diff
@@ -1,44 +1,86 @@
...
-    .catch(err => console.log(err));
-```
\ No newline at end of file
+await Storage.get('filename.txt', { expires: 60 })
+```
+
+
+### Frequently Asked Questions
+
+Users can run into unexpected issues, so we are giving you advance notice in documentation with links to open issues - please upvote what you need, to help the team prioritize.
```
Comment (sw-yx) [inline docs/lib/storage/fragments/js/download.md:77]:
```suggestion
Users can run into unexpected issues, so we are giving you advance notice in documentation with links to open issues - please vote for what you need, to help the team prioritize.
```

**Pull request merged**